src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -145,13 +166,78 @@ private void checkValue(@NonNull FormValidation formValidation) {
     }
 
     public boolean isDeleteArtifacts() {
-        return DELETE_ARTIFACTS;
+        return deleteArtifacts;
     }
 
     public boolean isDeleteStashes() {
-        return DELETE_STASHES;
+        return deleteStashes;
+    }
+    
+    public boolean getUsePathStyleUrl() {
+        return usePathStyleUrl;
+    }
+    
+    @DataBoundSetter
+    public void setUsePathStyleUrl(boolean usePathStyleUrl){
+        this.usePathStyleUrl = usePathStyleUrl;
+        save();
+    }
+    
+    public boolean getUseHttp() {
```

Comment:
java convention is usually `isUseHttp` for such method.

Reply:
It can go either way. Typically for `Describable`s we use `getXXX` even for `boolean` fields.